AGENTGATEWAY.md

```diff
@@ -0,0 +1,30 @@
+# Agentgateway Support in Istio
+
+Istio has **experimental** support for running the [agentgateway](https://github.com/agentgateway/agentgateway/) proxy as a dataplane. Agentgateway is a high-performance, lightweight proxy designed for cloud-native AI applications. Its primary control plane is the [kgateway](https://github.com/kgateway-dev/kgateway/) CNCF sandbox project, and the two together are fully conformant with the Gateway API. This document outlines the contract between the Istio and agentgateway projects and sets explicit milestones and advancement criteria for continued integration.
```

Comment:
Besides gateway api, does it has any other api? Can you point it if there is?    Just wanna to know how much effort and influence it require.